chart/newsfragments/32517.significant.rst

```diff
@@ -1,3 +1,3 @@
-Default Airflow image is updated to ``2.6.3``
+Default Airflow image is updated to ``2.7.0``
 
-The default Airflow image that is used with the Chart is now ``2.6.3``, previously it was ``2.6.2``.
+The default Airflow image that is used with the Chart is now ``2.7.0``, previously it was ``2.6.2``.
```

Comment:
```suggestion
The default Airflow image that is used with the Chart is now ``2.7.0``, previously it was ``2.6.3``.
```

Reply:
No. The previous release of the chart is still in 2.6.2 cc @jedcunningham